recipes/unidist/meta.yaml

```
@@ -0,0 +1,131 @@
+{% set name = "unidist" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5b2cc602604f75ed6a9323cd4cd0585a1c806d3465f50ecb7ff31772b80f2edb
+
+build:
+  # noarch disabled because the recipe now has selector for Ray, see below
+  # noarch: python
+  number: 0
+  skip: true  # [py<37]
+
+# the outputs map the unidist options on PyPI (see upstream definition
+# https://github.com/modin-project/unidist/blob/0.1.0/setup.py#L39-L55)
+# from "unidist[<option>]" to "unidist-<option>" as a conda-forge package
+outputs:
+  - name: unidist-all
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('unidist', exact=True) }}
+        - {{ pin_subpackage('unidist-dask', exact=True) }}
+        # ray does not have OSX builds yet
+        - {{ pin_subpackage('unidist-ray', exact=True) }}   # [not osx]
```

We should keep the mapping as on PyPI and this will lead to a solver error on osx instead of missing packages.
```suggestion
        - {{ pin_subpackage('unidist-ray', exact=True) }}
```